### ISSUE TITLE
Dumping option possibly dumps IR and TPLC of script wrapper instead of script itself

### ISSUE THREAD

**Issue by Renegatto** [Liqwid-Labs/plutus-extra#160]
Which seems not as expected behavior

I did not test that and concluded just from code, so may be wrong.
The following leads me to that conclusion.

# Investigation

Here we see that IR and TPLC uses `code :: CompiledCode a`, not `scr :: Script`.
It's a third argument of `DumpScript a` datatype.
```haskell
instance IsTest DumpScript where
  run opts (DumpScript nm scr code) _ = do
    runIf @DumpPlutusIR opts (dumpIR nm code)
    runIf @DumpPlutusTypedCore opts (dumpTypedCore nm code)
    runIf @DumpPlutusCore opts (dumpCore nm scr)
```
When search `DumpScript` on #136 changes, we see all constructions of this datatype, both in [tasty-plutus/src/Test/Tasty/Plutus/WithScript.hs](https://github.com/Liqwid-Labs/plutus-extra/pull/136/files#diff-f7cda392ced6c3e5276715ce4cac4390ad16d4785ab1302e66bd91e770ea9e52):

1. [At line 81](https://github.com/Liqwid-Labs/plutus-extra/pull/136/files#diff-f7cda392ced6c3e5276715ce4cac4390ad16d4785ab1302e66bd91e770ea9e52R81)
```haskell
  (DumpScript name (getValidator $ validatorScript val) wrap)
  ...
```
where `wrap :: CompiledCode (ValidatorType s -> WrappedValidatorType)`, so it's a wrapper

2. [At line 118](https://github.com/Liqwid-Labs/plutus-extra/pull/136/files#diff-f7cda392ced6c3e5276715ce4cac4390ad16d4785ab1302e66bd91e770ea9e52R118)
```
 dump = singleTest "Dump script" (DumpScript name (getMintingPolicy mp) wrap)
```
where `wrap ::  CompiledCode ((r -> ScriptContext -> Bool) -> (BuiltinData -> BuiltinData -> ()))`, and this is wrapper either

**Comment by re-xyr** [comment]
Oops, this looks like a careless mistake. Just fixed it in #136.

**Comment by re-xyr** [comment]
Closed by #136.
